app.py

```diff
@@ -0,0 +1,81 @@
+import os
+import requests
+from flask import Flask
+from flask import jsonify
+from flask import request
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+from utils import create_markdown_list
+from utils import generate_randomized_list
+from utils import generate_demo_order_blocks
+from utils import verify
+from utils import verify_is_valid_demo_day
+
+# channel_id = os.environ["SLACK_CHANNEL_ID"]
+slack_token = os.environ["SLACK_OAUTH_TOKEN"]
+sc = WebClient(token=slack_token)
+
+app = Flask(__name__)
+
+
+@app.route("/")
```

Comment:
is this route needed?

Reply:
I'll remove this and test it after demo...I don't wanna risk breaking it for tomorrow morning LOL but I kept it in there because I was under the impression Flask endpoints might break without a base route.

This might not be necessary since we're just interacting with this via Slack